src/components/BookContainer/BookContainer.js

```diff
@@ -6,9 +6,9 @@ import Book from '../Book/Book';
 
 const BookContainer = ({ books }) => (
   <section>
-    <ul>
+    <ul className="booklist">
       {books.map((book) => (
-        <li key={book.item_id}>
+        <li className="book" key={book.item_id}>
           <Book book={book} />
         </li>
       ))}
```

Comment:
- [ ] If you can check the design they are some features that you did not implemented yet, you should add them as a placeholder
![cmc](https://user-images.githubusercontent.com/44745855/150589982-21aec9e7-0cab-453a-9131-e1aca8839011.png)